static/api.js

```diff
@@ -1,5 +1,6 @@
-export default function api (url, data) {
+export default function api (url, data, userId) {
```

Comment:
I'd have called it something like `authValue` instead of `userId` for genericity purpose, but it's only used in one case here so tha's no big deal.